File: 5-Patron/php/Sprite.php
<?php
class Sprite{
	private $id;
	private $archivo;

	public function __construct($id, $archivo){
    $this->id = $id;
    $this->archivo = $archivo;
	}

	public function getId(){
		return $this->id;	
	}

	public function setId($id){
		$this->id = $id;
	}

	public function getArchivo(){
		return $this->archivo;	
	}

	public function setArchivo($archivo){
		$this->archivo = $archivo;
	}

	public function cargarSprite($file){
		$ruta = 'C:\CyberMagik2069\Sprites\Sprite.txt';
		echo 'Cargando archivo...';
		echo $ruta . ' detectada.';
	}
}
?>

File: 5-Patron/Java/Sprite.java
package patron5;

import java.io.File;

public class Sprite {
    private int id;
    private File file;

    public Sprite(int id, File file) {
        this.id = id;
        this.file = file;
    }

    public File getFile() {
        return file;
    }

    public int getId() {
        return id;
    }

    public void setFile(File file) {
        this.file = file;
    }

    public void setId(int id) {
        this.id = id;
    }
    
    public void cargarSprite(File file){
        String ruta = "C:\\CyberMagik2069\\Sprites\\Sprite.txt";
        System.out.println("Cargando arhcivo ......\n");
        System.out.println(ruta + " Detectada.");
    }
}
